lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
@ecgreb Thats true, while thinking about how to solve the exception I lost sight of that use case.

I think that:
1. Developers should be able to request location updates from a thread that hasn't called `Looper.prepare()` because they are able to do so with play services.
2. We could achieve this but it would require that we update the way we handle starting/stopping our service. Right now its a bound service but if we changed it to be an explicitly started/stopped service, it would continue to run in the background even after the calling application is killed. I remember we had this (in addition to the binding) but removed it...maybe we revisit adding it back?